pandas/core/style.py

```diff
@@ -255,6 +272,68 @@ def _translate(self):
                     precision=precision, table_styles=table_styles,
                     caption=caption, table_attributes=self.table_attributes)
 
+    def format(self, formatter, subset=None):
+        '''
```

Comment:
Can you use `"""` here instead of `'''`? Does not really matter of course, but I just like some uniformity on this :-) (and I think we mainly use the double ones for docstrings)